### ISSUE TITLE
This library fails to install on node versions greater than 0.7.0

### ISSUE THREAD

**Issue by atrivedi** [mvrilo/urban#10]
When trying to install, I get this error.
```
npm ERR! notsup Unsupported
npm ERR! notsup Not compatible with your version of node/npm: commander@0.3.0
npm ERR! notsup Required: {"node":">= 0.4.x < 0.7.0"}
npm ERR! notsup Actual:   {"npm":"3.3.3","node":"4.2.2"}
```

**Comment by mvrilo** [comment]
Fixed on b7b9926225

**Comment by atrivedi** [comment]
:+1: thank you!
